dask/dataframe/accessor.py

```diff
@@ -274,6 +274,9 @@ def _split(self, method, pat=None, n=-1, expand=False):
 
     @derived_from(pd.core.strings.StringMethods)
     def split(self, pat=None, n=-1, expand=False):
+        """
+        Known inconsistencies: ``expand=True`` with unknown ``n`` will raise a NotImplementedError.
+        """
```

Comment:
A couple of small suggestions:

- Move the content of the docstring to the same line as the initial `"""` in order avoid this portion being rendered as a block quote (see the screenshot below)
- Add double backticks to `NotImplementedError` so it uses code font

```suggestion
        """Known inconsistencies: ``expand=True`` with unknown ``n`` will raise a ``NotImplementedError``."""
```

<img width="1552" alt="Screen Shot 2022-06-09 at 2 54 25 PM" src="https://user-images.githubusercontent.com/11656932/172933259-df73c4c2-0445-4ffe-a7ad-cb0b75a5fad9.png">